### ISSUE TITLE
[PHP] Cant Use Guzzle PHP Generation

### ISSUE THREAD

**Issue by Robert430404** [swagger-api/swagger-codegen#5671]
##### Description

I'm wanting to use the template specified in PR #5190 and I can't find any documentation on how I go about using it. If you guys could point me in the right direction, I would really appreciate it!

##### Swagger-codegen version

2.3.0

##### Swagger declaration file content or url

n/a

##### Command line used for generation

java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i ../bookie-swagger.json -l php -o ../bookie-php-client

##### Steps to reproduce

n/a

##### Related issues

Not that I could find

##### Suggest a Fix

n/a

**Comment by wing328** [comment]
@Robert430404 the auto-generated PHP client comes with a README.md. Would that help?

**Comment by Robert430404** [comment]
I looked through all of that and it didn't answer my this particular question. If you look at the PR referenced, the template already exists and it's merged into the 2.3.0 branch of codegen, I just need to know the flag to use so it generates a guzzle client vs a curl client. I've tried using

* --library=guzzle
* --library=psr7
* --library=guzzlehttp
* etc...

And I looked through codegen to see if there were templates that I could reference in:

__( modules/swagger-codegen/src/main/resources/php )__

But didn't see anything that would help me.
